### ISSUE TITLE
Standalone usage?

### ISSUE THREAD

**Issue by topaxi** [webcomponents/shadycss#11]
Is it possible to use this without custom elements? It would be nice to have a custom css properties polyfill on its own :)

**Comment by kevinSuttle** [comment]
I was wondering the same about using this in React.

**Comment by cdbkr** [comment]
I totally agree, it would be great to use it with other approaches. I believe the "parseTemplate" function should be extended, because it seems to expect the styles declared in the element.
Just wondering if the css modules approach might be helpful for this case (exposing a function to accept the element and the scoped css).

**Comment by MikeKovarik** [comment]
+1

**Comment by sevenupcan** [comment]
I was wondering the same thing.

I have a few scripts which attach a shadow DOM to existing elements and I need a way to scope the styles included in the Shadow DOM.
